src/Command/Db/DbSizeCommand.php

```diff
@@ -361,7 +361,11 @@ private function getMySqlUsage(HostInterface $host, array $database) {
         $innoDbSize = 0;
         if ($allocatedSizeSupported) {
             $this->debug('Checking InnoDB separately for more accurate results...');
-            $innoDbSize = $host->runCommand($this->mysqlInnodbQuery($database));
+            try {
+                $innoDbSize = $host->runCommand($this->mysqlInnodbQuery($database));
+            }catch(Throwable $e) {//some PE clusters do not have the  PROCESS privilege(s) and thus, have no access to the sys_tablespaces, revert to legacy way 
```

Comment:
This catch should be more precise - the exception will be a `\Symfony\Component\Process\Exception\RuntimeException`, and the message will mention error code `1227`

Reply:
(and mainly, `Throwable` is a PHP 7 only thing, the CLI supports PHP 5.5.9+)

Reply:
You are absolutely right. Changed and pushed again.